lib/clientBuilder.js

```diff
@@ -159,7 +159,23 @@ proto.features.isTokenVerifySupported = function() {
 
 // { username, password, (relayState), (context) }
 proto.signIn = function (opts) {
-  return tx.postToTransaction(this, '/api/v1/authn', opts);
+  var sdk = this;
+  var body = opts || {};
+  return new Q(!body.sendFingerprint ? {} :
```

Comment:
would it be better that if we don't have the ```sendFingerprint``` option we just do what we were doing before?
```
if (!body.sendFingerprint) {
  return tx.postToTransaction(this, '/api/v1/authn', opts);
} else {
  return new Q(sdk.fingerprint()
...
}

Reply:
Doing it the way we have it now prevents me from have `tx.postToTransaction` twice

Reply:
You're right though, I think it'd be cleaner your way